lib/mayaUsd/python/wrapPrimUpdaterManager.cpp

```diff
@@ -35,6 +35,21 @@ PXR_NAMESPACE_USING_DIRECTIVE
 
 namespace {
 
+bool isAnimated(const std::string& nodeName)
+{
```

Comment:
Animation detection now needs to be done in Python/MEL instead of C++, so it is exposed to Python.